linux_os/guide/services/ssh/ssh_server/firewalld_sshd_port_enabled/oval/shared.xml

```diff
@@ -133,9 +133,10 @@
          OVAL resources in order to detect and assess only active zone, which are zones with at
          least one NIC assigned to it. Since it was possible to easily have the list of active
          zones, it was cumbersome to use that list in other OVAL objects without introduce a high
-         level of complexity to make sure environments with multiple NICs and multiple zones are
-         in use. So, in favor of simplicity and readbility it was decided to work with a static
-         list. It means that, in the future, it is possible this list needs to be updated. -->
+         level of complexity to ensure proper assessment in environments where multiple NICs and
+         multiple zones are in use. So, in favor of simplicity and readbility it was decided to
+         work with a static list. It means that, in the future, it is possible this list needs to
+         be updated. -->
```

Comment:
Nitpick: It is pretty difficult to identify what changed in these lines, :(

Propagating the breaklines down the paragraph makes it really hard to spot the diffs.
I'd rather have longer lines that are easier to diff.

Reply:
Very nitpick. hehe Even for a human sentence of 3-4 lines. I would rather prefer the style guide alignment in these cases.

Reply:
While the change is on 3 lines, I had to look back an forth on ever word on 6 lines.
It would be very easy to misread the diff. 

And the style guide should maintain our lives easy.

We have the one sentence per line for the markdown content.
And while this is a comment in an xml file, I think it would be nice to have a similar guidance for large blocks of comments.